Lib/traceback.py

```diff
@@ -1472,12 +1472,16 @@ def _compute_suggestion_error(exc_value, tb, wrong_name):
         obj = exc_value.obj
         try:
             d = dir(obj)
+            if wrong_name[:1] != '_':
```

Comment:
I guess you're using ``[:1]`` instead of ``[0]`` to handle empty strings. But there are no tests for this.

Reply:
It seems difficult to create tests for this with the current code.

If write `wrong_name[0]`, an IndexError raised for empty `wrong_name` will be caught by `except Exception`. If write `wrong_name[:1]`, then any non-empty names will be filtered out in the code below as too different from empty string. In any case the result is the same -- no suggestions.